eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -65,6 +68,12 @@ jobs:
       - DisplayName: "Test on MacOS"
         OSVmImage: "macOS-10.15"
         TestTargetFramework: netcoreapp2.1
+      - ${{ if eq(parameters.TestCanary, 'true') }}:          
+        - DisplayName: "Test on Windows for .Net Framework - Canary region"
+          OSVmImage: "windows-2019"
+          TestTargetFramework: net461
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)            
+          Location: centraluseuap
```

Comment:
Can you set this as a param with default value "centraluseuap"

Reply:
Done!